### PR TITLE
Add role and capability overview page

### PR DESCRIPTION
**Pull request by blackshadev**
Related to #200

## What has been done
- Overview page for roles + capabilities
- The menu was way to crowded, I opted to create a dropdown with bootstrap

## How to test

- Navigate to: "overig -> Rollen en Rechten"
- See role rights tables

## What needs to be done

- Add a better description to as a hover to the capabilities
- Add users which belong to certain roles (maxed to 5 persons)
- Maybe we need to think about displaying or searching. it is quite the amount of information.

### CODE REVIEW

**Comment by robinklaassen** [comment]
image

Not sure what I'm supposed to be seeing...

**Comment by blackshadev** [comment]
Did you run Yarn dev. Or something. You need to compile the sass files with
a command specified in package.json

**Comment by robinklaassen** [comment]
Jup, turns out I just ran `yarn install` and not `yarn dev`. Working now. I'll think about how the UI can be improved, because it's A LOT indeed.

**Comment by blackshadev** [comment]
Jup suggestions are welcome.
But this is more than we have, right.

**Comment by blackshadev** [comment]
Made it searchable. :D

File: resources/views/partials/nav.blade.php [hunk 1]
```diff
@@ -13,52 +13,61 @@
...
+								@can("viewAny", \App\Location::class)
+									<li class="{{ substr(Request::path(),0,9) == 'locations' ? 'active' : ''}}"><a href="{{ url('/locations') }}">Locaties</a></li>
+								@endcan
+
+								@role(["board"])
+								<li class="{{ substr(Request::path(),0,5) == 'event-packages' ? 'active' : ''}}">
+									<a href="{{ url('/event-packages') }}">Pakketten</a>
+								</li>
+								@endrole
+								<li class="{{ substr(Request::path(),0,6) == 'graphs' ? 'active' : ''}}"><a href="{{ url('/graphs') }}">Grafieken</a></li>
```
Comment (robinklaassen) [inline resources/views/partials/nav.blade.php:46]: Valt me nu op dat de Grafieken pagina niet onder een rol of capability valt. Maar ze zijn niet echt bedoeld voor deelnemers. Dus deze ook even onder de member role stoppen?

File: app/Http/Controllers/PagesController.php [hunk 2]
```diff
@@ -295,6 +295,10 @@ public function lists()
 	# Analytical graphs
 	public function graphs()
 	{
+        if (!\Auth::user()->hasRole("member")) {
+			abort(403, 'Onvoldoende rechten om lijsten in te zien');
```
Comment (robinklaassen) [inline app/Http/Controllers/PagesController.php:299]:
```suggestion
			abort(403, 'Onvoldoende rechten om grafieken in te zien');
```

**Pull request merged**